rosidl_typesupport_opensplice_c/resource/msg__type_support_c.cpp.em

```diff
@@ -397,7 +397,7 @@ else:
       return "failed to assign string into field '@(field.name)'";
     }
 @[  elif field.type.is_primitive_type()]@
-    ros_message->@(field.name) = dds_message->@(field.name)_@(' == TRUE' if field.type.type == 'bool' else '');
+    ros_message->@(field.name) = @('((' if field.type.type == 'bool' else '')dds_message->@(field.name)_@(')? true : false)' if field.type.type == 'bool' else '');
```

Comment:
I haven't tried to run this line yet but it looks "fishy" :wink: Can you describe what it is supposed to do since I can't decipher the template?

Reply:
Have to get back to you on that one, as I'm not the one who made it

Reply:
btw cannot be run on windows without #197

Reply:
I will trigger a new build once #197 passed CI and has been merged.